Neo4j.Driver/Neo4j.Driver.Tests.Integration/Stress/CausalClusterStressTests.cs

```diff
@@ -146,6 +152,42 @@ private ClusterAddresses DiscoverClusterAddresses()
             return new ClusterAddresses(followers, readReplicas);
         }
 
+        private bool UsingBoltMoreThan5_0()
+        {
+            return _driver.GetServerInfoAsync().GetAwaiter().GetResult().ProtocolVersion
```

Comment:
This makes me think we should have an internal method in the driver to do this :)